core/micro-services/workflow-compiling-service/src/main/scala/edu/uci/ics/texera/service/CORSFilter.scala

```diff
@@ -0,0 +1,23 @@
+package edu.uci.ics.texera.service
+
+import jakarta.ws.rs.container.{
+  ContainerRequestContext,
+  ContainerResponseContext,
+  ContainerResponseFilter
+}
+import jakarta.ws.rs.core.MultivaluedMap
+import jakarta.ws.rs.ext.Provider
+
+@Provider
+class CORSFilter extends ContainerResponseFilter {
```

Comment:
I am still concerned about disabling CORS check. This is also related to how development would work. I think we need to let the frontend talk to a single gateway, and the gateway to forward requests to other services. Please leave a to-do here and explain our current decision and future plan.

Reply:
CORs is removed now